### ISSUE TITLE
JCache should be part of Java EE 8

### ISSUE THREAD

**Issue by OndroMih** [Adopt-a-JSR/jcache-javaee#3]
JCache was always intended to be part of Java EE as you may have read, but somehow it missed the boat for inclusion in EE 7, and now that it could be added to EE 8 it just doesn't happen, despite various people at some time actively working on this.

JCache could be used in other JSRs, like in JSF for Facelet Cache, or in JPA for 2nd level cache. But that requires that JCache is officially included in Java EE 8 JSR. The latest JSR draft still does not include it.

**Comment by OndroMih** [comment]
Probably an additional JSR for JCache is needed, to bridge the gap between the current spec and Java EE. It is unlikely that current JCache spec leads would be interested to lead a new version of the JSR, therefore the questions are: 
- who would be willing to lead a new JSR?
- if it is realistic to transfer the intellectual properties from current spec leads to new ones, or it is better/possible to submit a completely new separate JSR?

**Comment by eolivelli** [comment]
Which are the steps to start a new JSR? How many people or companies are needed? 
My company can give  support to this new JSR.
This repo is a good starting point

**Comment by eolivelli** [comment]
Which are the steps to start a new JSR? How many people or companies are needed? 
My company can give  support to this new JSR.
This repo is a good starting point

**Comment by OndroMih** [comment]
Thanks for the support, @eolivelli . Is your company a member of the JCP? Would you join an EG in case we need to file a new JSR?

**Comment by eolivelli** [comment]
Yes. My company Diennea is a JCP and I am the primary contact. We will support the new JSR and join the EG
